cleanup-itest-targets.sh

```diff
@@ -0,0 +1,5 @@
+#!/bin/bash
+
+set -e
+
+rm -rf ./judo-ui-react-itest/*/*/target
```

Comment:
I suggest replacing . With $0

Reply:
No idea what that does. I want all sub-folders to be cleared.